src/sortStudents.ts

```diff
@@ -1,16 +1,53 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'averageAge'
 }
 
 // create SortOrder type
-export type SortOrder;
+export enum SortOrder {
+  asc = 'asc',
+  desc = 'desc',
+}
 
+// eslint-disable-next-line no-multiple-empty-lines
 
-export function sortStudents(students, sortBy, order) {
-  // write your function
+// eslint-disable-next-line max-len
+export function sortStudents(students: Student, sortBy: SortType, order: SortOrder): [] {
+  if (order === SortOrder.asc) {
+    switch (sortBy) {
+      case SortType.Name || SortType.Surname:
```

Comment:
```suggestion
      case SortType.Name:
      case SortType.Surname:
```
better use such syntax